analyzer/tests/functional/ctu_failure/test_ctu_failure.py

```diff
@@ -114,7 +114,7 @@ def test_ctu_on_demand_logs_ast_import(self):
         """ Test that Clang indeed logs the AST import events when using
         on-demand mode.
         """
-        self.__set_up_test_dir('ctu_on_demand_failure')
+        self.__set_up_test_dir('ctu_failure')
```

Comment:
Why do we have to change the test dir?

Reply:
There is no such project in the tests like ctu_on_demand_failure. See: https://github.com/Ericsson/codechecker/tree/master/analyzer/tests/projects